pyproject.toml

```diff
@@ -25,8 +25,24 @@ keywords=['interactive brokers', 'rest api', 'python api', 'ibkr python api', 'i
 [project.urls]
 Homepage = "https://github.com/Voyz/ibind"
 
+[tool.ruff.lint]
+# Defined rules here:
+# https://docs.astral.sh/ruff/rules/
+select = [
+    "F", # pyflakes
+    "E", # pycodestyle - Errors
+    "W", # pycodestyle - Warnings
+    "S", # Security
+    "N", # pep8-naming
+]
+
+ignore = [
+    "E501",  # Ignore line length errors
```

Comment:
I wanna bring up ignoring 'E712' - since in several locations we do care about doing `if x==False` rather than `if not x`, seeing that x can be None. Any objections?

Reply:
Nope.   I actually was going to consider adding 2-3 other rulesets that I think are helpful for code quality, but I know we'll be opinionated and want to ignore several of the rules.  Do I mind I add E712, and add the additional rulesets so we can see what we want to ignore?

Reply:
Great, what other rulesets you think of adding?

Reply:
```
[
"PL", # pylint
"DOC", # Documentation
"D", # pydocstyle
]
```

The findings could end up being pretty pedantic, more out of curiosity if we end up getting new findings that we actually see value in.

Reply:
This is resolved now?

Reply:
Not sure exactly what you're referring to regarding resolved.  We have added several rule sets, and accompanying ignores that we either don't find value, or we're punting on for future refactors.